web/core/src/jsMain/kotlin/org/jetbrains/compose/web/css/properties/border.kt

```diff
@@ -83,26 +83,115 @@ fun StyleScope.borderWidth(width: CSSNumeric) {
+            "Despite that, it behaves correctly (see https://developer.mozilla.org/en-US/docs/Web/CSS/border-width)." +
+            "Therefore this function still can be used without specifying parameter names. Replacement is not necessary." +
+            " It'll remain Deprecated until 2.0, and in 2.0 parameter names should be changed to: " +
+            "`vertical: CSSNumeric` and `horizontal: CSSNumeric`",
+    replaceWith = ReplaceWith(
+        expression = "borderWidth2(vertical = topLeft, horizontal = bottomRight)"
+    )
+)
 fun StyleScope.borderWidth(topLeft: CSSNumeric, bottomRight: CSSNumeric) {
-    property("border-width", "$topLeft $bottomRight")
+    borderWidth2(topLeft, bottomRight)
+}
+
+/**
+ * This function is a temporary replacement for `borderWidth(topLeft: CSSNumeric, bottomRight: CSSNumeric)`.
+ * After a fix for parameters names in `borderWidth`,
+ * this function will be Deprecated with a replaceWith `borderWidth`. Later `borderWidth2` will be Removed.
+ * @see borderWidth
+ */
+fun StyleScope.borderWidth2(vertical: CSSNumeric, horizontal: CSSNumeric) {
+    property("border-width", "$vertical $horizontal")
 }
 
+
+/**
+ * https://developer.mozilla.org/en-US/docs/Web/CSS/border-width
+ *
+ * First argument sets the top border width.
+ * Second argument sets the horizontal borders width.
+ * Third argument sets the bottom border width.
+ *
+ * NOTE: Temporary deprecation until 2.0 because of wrong parameters names.
+ * In 2.0 the parameter names should be fixed. Despite the wrong parameter names, the function behaves correctly.
+ */
+@Deprecated(
+    message = "This function has misleading parameter names. " +
+            "Despite that, it behaves correctly (see https://developer.mozilla.org/en-US/docs/Web/CSS/border-width)." +
+            "Therefore this function still can be used without specifying parameter names. Replacement is not necessary." +
+            " It'll remain Deprecated until 2.0, and in 2.0 parameter names should be changed to: " +
+            "`top: CSSNumeric`, `horizontal: CSSNumeric`, `bottom: CSSNumeric`",
+    replaceWith = ReplaceWith(
+        expression = "borderWidth3(top = topLeft, horizontal = topRightAndBottomLeft, bottom = bottomRight)"
+    )
+)
 fun StyleScope.borderWidth(
     topLeft: CSSNumeric,
     topRightAndBottomLeft: CSSNumeric,
     bottomRight: CSSNumeric
 ) {
-    property("border-width", "$topLeft $topRightAndBottomLeft $bottomRight")
+    borderWidth3(topLeft, topRightAndBottomLeft, bottomRight)
+}
+
+/**
+ * This function is a temporary replacement for `borderWidth(topLeft: CSSNumeric, topRightAndBottomLeft: CSSNumeric, bottomRight: CSSNumeric)`.
+ * After a fix for parameters names in `borderWidth`,
+ * this function will be Deprecated with a replaceWith `borderWidth`. Later `borderWidth3` will be Removed.
+ * @see borderWidth
+ */
+fun StyleScope.borderWidth3(top: CSSNumeric, horizontal: CSSNumeric, bottom: CSSNumeric) {
```

Comment:
I decided to add 3 new functions: 
borderWidth2 borderWidth3 and borderWidth 4 to note the number of parameters.
I didn't want to add one new function and overload it for different parameters count, because I didn't see a good function name for it. So the number just notes the number of parameters

Reply:
But when we undeprecate the original function, it will be the single function, without suffix. So maybe better to make API closer to the new API?

2/3/4 as suffix for number of arguments isn't obvious :)

Reply:
Okay. Then I think the number suffix won't make any sense at all.
Maybe borderWidthTemp? Meaning temporary?